### PR TITLE
feat: onboarding page

### PR DESCRIPTION
**Pull request by darkskygit**

### CODE REVIEW

**Comment by JimmFly** [comment]
Current dependencies on/for this PR:
* `canary`
  * **PR #5277** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5277?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #5327** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5327?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5277?utm_source=stack-comment).

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    },
+    []
+  );
+
+  const handleInputChange = useCallback((id: string, value: string) => {
+    setInputValues(prev => ({ ...prev, [id]: value }));
+  }, []);
+
+  useEffect(() => {
+    // const fetchQuestions = async () => {
```
Comment (Brooooooklyn) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx]: remove these lines

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+          answer: questionAnswers,
+        };
+      }
+    );
+
+    setQuestionIdx(prevIdx => prevIdx + 1);
+
+    console.log('finalAnswers:', finalAnswers);
+
+    // try {
```
Comment (Brooooooklyn) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx]: remove these lines

File: packages/frontend/component/src/components/auth-components/onboarding-page.tsx [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+          form: user.id,
+          ask: question.id || 'undefined-question-id',
+          answer: questionAnswers,
+        };
+      }
+    );
+
+    setQuestionIdx(prevIdx => prevIdx + 1);
+
+    console.log('finalAnswers:', finalAnswers);
```
Comment (Brooooooklyn) [inline packages/frontend/component/src/components/auth-components/onboarding-page.tsx]: remove

**Review by Brooooooklyn — APPROVED** [review]

**Pull request merged**